### ISSUE TITLE
onCentered event

### ISSUE THREAD

**Issue by emitategh** [SubZane/simplegmaps#29]
Is it posible to add onCentered event? Would it be the same to use onPlaceChanged?

**Comment by SubZane** [comment]
yes, or at least I think so (without checking the code) you can add hooks anywhere in the code literary :)

I don't have time right now to do an update, but if you check the code, you can mod it yourself and add the hooks you want
